docs/learn/universal-profile/key-manager/execute-relay-transactions.md

```diff
@@ -8,29 +8,41 @@ import TabItem from '@theme/TabItem';
+As an example for this guide, the transaction that will be executed without requiring to pay for gas will be a LYX transfer.
 
-Another example would be Alice sending an encoded transaction that updates the [LSP3Profile](../../../standards/universal-profile/lsp3-profile-metadata.md) picture of her [Universal Profile](../../../standards/universal-profile/introduction.md) or brand to a second user, Bob, who executes the transaction and pays the gas cost of the transaction on behalf of Alice.
+![Normal vs Gas-Less Transactions](/img/standards/lsp6/lsp6-direct-vs-relay-execution.jpeg)
 
-To execute the transaction, the executing party needs to know:
+## What are Relay Transactions?
 
-- the encoded ABI of the transaction that will get executed,
-- the transaction signature,
-- the nonce of the key that signed the transaction.
+The [LSP6 KeyManager](../../../standards/universal-profile/lsp6-key-manager.md) standard empowers any user to execute a transaction on behalf of a [Universal Profile](../../../standards/universal-profile/introduction.md), provided they possess a valid transaction signed by a key that controls the Universal Profile. This capability significantly enhances the user's control and flexibility in managing their transactions.
 
-The transaction is then executed via the [`executeRelayCall`](../../../contracts/contracts/LSP6KeyManager/LSP6KeyManager.md#executerelaycall) function of the [LSP6 Key Manager](../../../standards/universal-profile/lsp6-key-manager.md).
+Relayed execution enables the creation of [Transaction Relayer Services](../../../standards/relayer-api.md), which can execute transactions on behalf of users, _without having any control over their Universal Profile_. Relayers abstract away the gas from users.
 
-## Setup
+Another example use case would be Alice sending an encoded transaction that updates the LSP3Profile picture of her Universal Profile to a second user, Bob, who executes the transaction and pays the gas cost on behalf of Alice.
+
+:::success Did you know?
 
-First, the transaction to be executed by a third party has to be prepared. We will prepare an [`executeRelayCall`](../../../contracts/contracts/LSP6KeyManager/LSP6KeyManager.md#executerelaycall) transaction to be executed by a third party. This logic can be implemented _client-side_ and then sent to a _third-party_ application or service, such as a _Transaction Relay Service_.
+Users who created their 🆙 via [_universalprofile.cloud_](https://universalprofile.cloud) **benefit from a monthly gas quota paid by LUKSO**. ⛽️ ❌
```

Comment:
Why the ❌ emoji? It looks like it's something bad but actually it s great that users are being subsidized